### PR TITLE
Feature/angular12 attempt

### PR DESCRIPTION
**Pull request by garyluu**
Update to Angular 12. Probably going to update a lot of other random things later.

This should also fix https://ucsc-cgl.atlassian.net/browse/SEAB-2524

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,26 +33,26 @@
...
-    "@angular/platform-browser-dynamic": "^11.2.14",
-    "@angular/platform-server": "^11.2.14",
-    "@angular/router": "^11.2.14",
-    "@datorama/akita": "^6.1.3",
+    "@angular/animations": "^12.0.4",
+    "@angular/cdk": "^12.0.4",
+    "@angular/common": "^12.0.4",
+    "@angular/compiler": "^12.0.4",
+    "@angular/core": "^12.0.4",
+    "@angular/flex-layout": "^12.0.0-beta.34",
```
Comment (coverbeck) [inline package.json:41]: Just looked, and this has been in beta for 4 years!

**Review by coverbeck — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (denis-yuen) [reply package.json:41]: :grimacing:

Comment (garyluu) [reply package.json:41]: yep

**Comment by NatalieEO** [comment]
Does the new high severity issue this introduces apply to us? https://app.snyk.io/org/denis-yuen/project/2fb26a6b-c753-4608-b78e-95b808dfb51d/pr-check/39712553-501d-46b5-bbb0-f539a3bd18a0

**Comment by garyluu** [comment]
> Does the new high severity issue this introduces apply to us? https://app.snyk.io/org/denis-yuen/project/2fb26a6b-c753-4608-b78e-95b808dfb51d/pr-check/39712553-501d-46b5-bbb0-f539a3bd18a0

It fixes ajv one and adds one. I think it's fairly ini will get fixed before the next release

File: angular.json [hunk 2]
```diff
@@ -47,7 +46,13 @@
               "node_modules/ace-builds/src-min-noconflict/theme-idle_fingers.js",
               "node_modules/ace-builds/src-min-noconflict/ext-searchbox.js",
               "node_modules/bootstrap/dist/js/bootstrap.js"
-            ]
+            ],
+            "vendorChunk": true,
+            "extractLicenses": false,
```
[the remark this answers is not in the document]
Comment (garyluu) [reply angular.json:51]: Interesting...took a look and the Angular generated one has a similar file length but is much more verbose (often printing out the entire license).  The Angular one also includes some transitive deps so I'm not sure what's different.

**Review by NatalieEO — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (denis-yuen) [reply package.json:41]: On the other hand, gmail did five years https://slate.com/news-and-politics/2009/07/why-google-kept-gmail-in-beta-for-so-many-years.html

**Pull request merged**